incompat.css

```diff
@@ -0,0 +1,22 @@
+p {
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    transition: none;
+    appearance: textfield;
+    appearance: textfield;
+    appearance: textfield;
```

Comment:
CSS Appearance not supported by: IE (10,11), Edge (12,13), Firefox (42,43), Chrome (46,47), Safari (8,9), Opera (33,34), iOS Safari (8.1-8.4,9.0-9.2), IE Mobile (10,11)